src/metabase/models/setting.clj

```diff
@@ -274,6 +271,9 @@
    ;; where this setting should be visible (default: :admin)
    [:visibility Visibility]
 
+   ;; should this setting be encrypted `:never` or `:maybe` (when `MB_ENCRYPTION_SECRET_KEY` is set)
+   [:encryption [:enum :never :maybe]]
```

Comment:
The default is (basically) `:maybe`, right? Might be a nice thing to add in the comment or some other docs